test/com/yetanalytics/lrs_admin_ui/functions/time_test.cljs

```diff
@@ -0,0 +1,20 @@
+(ns com.yetanalytics.lrs-admin-ui.functions.time-test
+  "Test time utility functions. Note that any headless browser running this
+  should have TZ=America/New_York set."
+  (:require [clojure.test :refer-macros [deftest is testing]]
+            [com.yetanalytics.lrs-admin-ui.functions.time
+             :refer [tz-offset-mins*
+                     utc->local-datetime]]))
+
+(deftest tz-offset-mins*-test
+  (testing "yields different offsets for different dates"
+    (is (not=
+         (tz-offset-mins* (new js/Date "2025-03-04T00:00:00Z"))
+         (tz-offset-mins* (new js/Date "2025-03-19T00:00:00Z"))))))
+
+(deftest utc->local-datetime-test
```

Comment:
We should also add tests for `local-datetime->utc`

Reply:
so help me glob [f9f00b7](https://github.com/yetanalytics/lrs-admin-ui/pull/109/commits/f9f00b78dca322d5185773c7161aa641cb15e938)